src/ResourceManager/Automation/Commands.Automation/help/Import-AzureRMAutomationRunbook.md

```diff
@@ -23,7 +23,7 @@ Import-AzureRmAutomationRunbook [-Path] <String> [-Description <String>] [-Name
 ## DESCRIPTION
 The **Import-AzureRmAutomationRunbook** cmdlet imports an Azure Automation runbook. Specify the
 path to a wps_2 script (.ps1 ) file to import for wps_2 and wps_2 Workflow runbooks, or to a
-graphical runbook (.graphrunbook) file for graphical runbooks. The name of the file becomes the
+graphical runbook (.graphrunbook) file for graphical runbooks. (.py) files are required when importing python2 scripts The name of the file becomes the
```

Comment:
Sorry for being picky, punctuation seems weird here. Also, I believe the extension requirement is true for other runbook type too no? Also can you remove the extra space from (.ps1 ) please. 

Suggestion : 
"Specify the path to a wps_2 script (.ps1) file to import for wps_2 and wps_2 Workflow runbooks, or to a graphical runbook (.graphrunbook) file for graphical runbooks, or to a python 2 script (.py) file for python 2 runbooks... "

Thoughts?